bin/start.js

```diff
@@ -34,7 +40,14 @@ browserSync.init({
                 `${CONFIG.sourceDir}/**/*`
             ],
             fn: () => {
-                Build.buildSync();
+                if (timeSinceDebounce(devDebounceMarker) > 5) {
+                    const lint = exec('yarn lint');
+                    lint.stderr.pipe(process.stderr);
```

Comment:
Linting is not necessary here. This is being handled by the `eslint-loader` in the webpack configuration. The `yarn lint` command is for CI/CD purposes to determine whether or not a build passess (if linting should fail in CI/CD, the build should fail).